src/anyon/anyon.jl

```diff
@@ -23,28 +23,49 @@ Quantum Processing Unit:
 struct AnyonYukonQPU <: AbstractQPU
     client                  ::Client
     status_request_throttle ::Function
+    connectivity            ::AbstractConnectivity
 
-    AnyonYukonQPU(client::Client; status_request_throttle=default_status_request_throttle) = new(client, status_request_throttle)
-    AnyonYukonQPU(; host::String, user::String, access_token::String, status_request_throttle=default_status_request_throttle) = new(Client(host=host, user=user, access_token=access_token), status_request_throttle)
+    AnyonYukonQPU(client::Client; status_request_throttle=default_status_request_throttle) = new(client, status_request_throttle,LineConnectivity(6))
+    AnyonYukonQPU(; host::String, user::String, access_token::String, status_request_throttle=default_status_request_throttle) = new(Client(host=host, user=user, access_token=access_token), status_request_throttle,LineConnectivity(6))
 end
 
-const line_connectivity_label="linear"
 
-get_metadata(::AnyonYukonQPU) = Dict{String,Union{String,Int}}(
+get_metadata(qpu::AnyonYukonQPU) = Dict{String,Union{String,Int}}(
     "manufacturer"  =>"Anyon Systems Inc.",
     "generation"    =>"Yukon",
     "serial_number" =>"ANYK202201",
-    "qubit_count"   =>6,
-    "connectivity_type"  =>line_connectivity_label
+    "qubit_count"   =>get_num_qubits(qpu.connectivity),
+    "connectivity_type"  =>get_connectivity_label(qpu.connectivity)
 )
 
-get_client(qpu_service::AnyonYukonQPU)=qpu_service.client
+struct AnyonMonarqQPU <: AbstractQPU
+    client                  ::Client
+    status_request_throttle ::Function
+    connectivity            ::AbstractConnectivity
+
+    AnyonMonarqQPU(client::Client; status_request_throttle=default_status_request_throttle) = new(client, status_request_throttle,LatticeConnectivity(4,3))
+    AnyonMonarqQPU(; host::String, user::String, access_token::String, status_request_throttle=default_status_request_throttle) = new(Client(host=host, user=user, access_token=access_token), status_request_throttle,LatticeConnectivity(4,3))
+end
+
+get_metadata(qpu::AnyonMonarqQPU) = Dict{String,Union{String,Int}}(
+    "manufacturer"  =>"Anyon Systems Inc.",
+    "generation"    =>"MonarQ",
+    "serial_number" =>"ANYK202301",
+    "qubit_count"   =>get_num_qubits(qpu.connectivity),
+    "connectivity_type"  =>get_connectivity_label(qpu.connectivity)
+)
 
-get_num_qubits(qpu::AnyonYukonQPU)=get_metadata(qpu)["qubit_count"]
+get_client(qpu_service::AbstractQPU)=qpu_service.client
 
-print_connectivity(qpu::AnyonYukonQPU,io::IO=stdout)=println(io, "1──2──3──4──5──6")
+UnionAnyonQPU=Union{AnyonYukonQPU,AnyonMonarqQPU}
 
-function Base.show(io::IO, qpu::AnyonYukonQPU)
+get_num_qubits(qpu::UnionAnyonQPU)=get_num_qubits(qpu.connectivity)
+
+get_connectivity(qpu::UnionAnyonQPU) = qpu.connectivity
+
+print_connectivity(qpu::AbstractQPU,io::IO=stdout)=print_connectivity(qpu.connectivity,Int[],io)
```

Comment:
`AbstractQPU` does not have a `qpu.connectivity` member. I think this function should be implemented for `UnionAnyonQPU` and use `get_connectivity` to get the connectivity.

Reply:
I changed it to `get_connectivity()` but left the type to `AbstractQPU` because a VirtualQPU also has a AllToAllConnectivity. get_connectivity is part of the AbstractQPU's interface.